src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -131,6 +133,22 @@ class JesBackend extends Backend with LazyLogging {
     JesOutput(value, engineFunctions.gcsPathFromAnyString(value).toString, Paths.get(value))
   }
 
+  override def stdout(workflowId: WorkflowId, callFqn: FullyQualifiedName): WdlFile = {
+    val fqnParts = callFqn.split("\\.")
```

Comment:
Also this might be a place where a proper class would add real value over the `FullyQualifiedName` type alias in checking this constraint at construction time.